docs/knowledge_base/issue_tags.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Issue Tags
+category: 6255e67693d5200013b1fa41
+hidden: false
+---
+
+## Issue Tags
+
+Phylum uses tags to uniquely identify issues raised by the system. These tags can be used in custom [policy](https://docs.phylum.io/docs/policy) creation.
+
+- Severities
+  - Minimum (`I`)
+  - Medium (`M`)
+  - High (`H`)
+  - Critical (`C`)
+  - Situation-dependent severity is represented in the table by `.`
+- Domains
+  - Author Risk (`A`)
+  - Engineering Risk (`E`)
+  - Malicious Code (`M`)
+  - Vulnerabilities (`V`)
+  - License Risk (`L`)
+    
+
+| Tag ID | Issue Name | Issue Description |
+| --- | --- | --- |
+| CA0001    | [Bad Author](https://docs.phylum.io/docs/bad_author) | Author is known malicious |
+| CM0001    | [IP Detection](https://docs.phylum.io/docs/ip_identification) | Package contains suspicious IP addresses |
+| CM0003    | Landing Binary | Package is known to contain malware |
```

Comment:
This isn't correct. Landing binary is just that the package uses LOL binaries. Maybe update to say "Package is using LOL binaries in a malicious way"?

Reply:
@louislang I got that straight from the issue text: https://github.com/phylum-dev/rules/blob/06fe2e10e8726ac2b663aef3b59cf0688e4b180e/src/rules/landing_binary_rule.py#L20

Shall I update the issue text to say, "Package is using LOL binaries in a malicious way"?

Reply:
oh, I think I see. That's some old stuff from super _super_ early Phylum days: https://github.com/phylum-dev/rules/blob/06fe2e10e8726ac2b663aef3b59cf0688e4b180e/src/rules/landing_binary_rule.py#L14

I think this is overcome by the curated malware stuff. I'd leave it and update the text like you said, but in practice idk that anyone will run into it.

Reply:
Final answer - how do you want this updated?

Reply:
"Package is using living off the land binaries in a known malicious way"